site/en/docs/extensions/reference/declarativeWebRequest/index.md

```diff
@@ -31,13 +31,13 @@ you want to trigger a message.
 
 All other actions require host permissions to all URLs.
 
-As an example, if `"*://*.google.com/*"` is the only host permission an extension has, then such an
+As an example, if `"https://*.google.com/*"` is the only host permission an extension has, then such an
```

Comment:
In this case the original use of `*://` may be preferable as it's shorthand for both the HTTP and HTTPS protocols.

Reply:
I disagree; the examples immediately below all work perfectly fine with https://. (I do appreciate your in-depth analysis of this PR though!)